ptx/S_12_3_ParamLineIntegrals.ptx

```diff
@@ -2,427 +2,264 @@
+            <answer> <m>0 </m></answer>
+      </task>
 
-          <li>
+      <task xml:id="A_12_3_2_part2">
             <p>
-              Let <m>C_2</m> be the line segment from <m>(1,0)</m> to
-              <m>(-1,0)</m>.
-              Calculate <m>\int_{C_2}\vF\cdot d\vr</m>.
+              Let <m>C_2</m> be the line segment from <m>(1,0)</m> to <m>(-1,0)</m>. Calculate <m>\int_{C_2}\vF\cdot d\vr</m>.
             </p>
-          </li>
+            <solution><p>In this part, the curve <m>C_2</m> is traced out by <m>\vr(t) =\langle -t,0 \rangle</m> as <m>-1 \leq t\leq 1</m>. So <me> \int_{C_2} \vF\cdot d\vr =\int_{-1}^1 \langle -t,0\rangle \cdot \langle-1,0\rangle \enspace dt = 0</me> </p>
+            </solution>
+            <answer> <m>0</m></answer>
+      </task>
 
-          <li>
+      <task xml:is="A_12_3_2_part3">
             <p>
-              Let <m>C_3</m> be the circle of radius <m>3</m> centered at the origin,
-              oriented counterclockwise.
-              Calculate <m>\oint_{C_3} \vF\cdot d\vr</m>.
+              Let <m>C_3</m> be the circle of radius <m>3</m> centered at the origin, oriented counterclockwise. Calculate <m>\oint_{C_3} \vF\cdot d\vr</m>.
             </p>
-          </li>
-        </ol>
+            <solution> <p>In this part, the curve <m>C_3</m> is traced out by <m>\vr(t) =\langle 3\cos(t),3\sin(t) \rangle</m> as <m>0 \leq t\leq 2\pi</m>. So <me> \int_{C_3} \vF\cdot d\vr =\int_{0}^{2\pi} \langle 9(\sin(t))^2,2\cos(t)\sin(t)+3 \rangle \cdot \langle -3\sin(t),3\cos(t) \rangle \enspace dt = 0</me> </p>
+            </solution>
+            <answer><m>0</m> </answer>
+      </task>
       </p>
     </activity>
   </subsection>
 
   <subsection>
     <title>Alternative Notation for Line Integrals</title>
     <p>
-      In contexts where the fact that the quantity we are measuring via a
-      line integral is best measured via a dot product
-      (such as calculating
-      work),
-      the notation we have used thus far for line integrals is fairly
-      common.
-      However, sometimes the vector field is such that the units on
-      <m>x</m>, <m>y</m>, and <m>z</m> are not distances.
-      In this case, a dot product may
-      not have a physical meaning, and an alternative notation using
-      differentials can be common.
-      Specifically, if <m>\vF(x,y,z) =
-      F_1(x,y,z)\vi + F_2(x,y,z)\vj + F_3(x,y,z)\vk</m>, then
+      In contexts where the fact that the quantity we are measuring via a line integral is best measured via a dot product (such as calculating work), the notation we have used thus far for line integrals is fairly common. However, sometimes the vector field is such that the units on <m>x</m>, <m>y</m>, and <m>z</m> are not distances. In this case, a dot product may not have a physical meaning, and an alternative notation using differentials can be common. Specifically, if <m>\vF(x,y,z) = F_1(x,y,z)\vi + F_2(x,y,z)\vj + F_3(x,y,z)\vk</m>, then
```

Comment:
In general, try not to re-wrap paragraphs. There's a method to David F's madness for pretty-printing in that it makes it easier to read these diffs and identify what changed.